components/inputs/input-date.js

```diff
@@ -248,16 +254,10 @@ class InputDate extends FormElementMixin(LocalizeCoreElement(LitElement)) {
 		if (this._textInput) this._textInput.focus();
 	}
 
-	async validate(validationType) {
-		let childErrors = [];
-		const inputTextElem = this.shadowRoot.querySelector('d2l-input-text');
-		if (inputTextElem) {
-			await inputTextElem.updateComplete;
-			childErrors = await Promise.resolve(inputTextElem.validate(validationType));
-			this._inputTextInvalid = inputTextElem.invalid;
```

Comment:
So I think this still needs to be set, otherwise the icon doesn't change if `required` and no entry

Reply:
Good catch, I'll add `this.invalid || this.childErrors.size > 0` so the error icon shows if the `text-input` is invalid.